.circleci/bin/train

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env bash
+
+[[ "$TRACE" ]] && set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# GIT_REPO="trade-tariff-lambdas-fpo-search"
```

Comment:
This was just me playing locally. I'll populate this in a bit with the latest version of the AMI.

My plan was to ssh on with the key pair and execute this script as the ec2-user